### PR TITLE
new: fnv32hash helper

### PR DESCRIPTION
**Pull request by sibicramesh**
--> hash helper function

### CODE REVIEW

**Comment by sibicramesh** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "host-service-name",
    "component": "enforcerd",
    "pr-id": "1303",
    "commit-sha": "33821d66e8ed1e3d385520fae8bf384d55769250"
  },
  {
    "project": "host-service-name",
    "component": "trireme-lib",
    "pr-id": "839",
    "commit-sha": "4bf2c6521228d0f7089fba3d0f77783f30fc2238"
  },
  {
    "project": "host-service-name",
    "component": "gaia",
    "pr-id": "371",
    "commit-sha": "c927dd3456a09355987c70346f6ce9a0f9aed212"
  }
]
```

**Comment by sibicramesh** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "host-service-name",
    "component": "enforcerd",
    "pr-id": "1303",
    "commit-sha": "cf9a2e3e33994b266ff44f850b8b7cbe611ef48f"
  },
  {
    "project": "host-service-name",
    "component": "trireme-lib",
    "pr-id": "839",
    "commit-sha": "4bf2c6521228d0f7089fba3d0f77783f30fc2238"
  },
  {
    "project": "host-service-name",
    "component": "gaia",
    "pr-id": "371",
    "commit-sha": "c927dd3456a09355987c70346f6ce9a0f9aed212"
  }
]
```

File: policy/types.go [hunk 1]
```diff
@@ -365,3 +367,23 @@ type RuntimeError struct {
...
+		aggregatedData += ed
+	}
+
+	hash := fnv.New32()
+	if _, err := hash.Write([]byte(aggregatedData)); err != nil {
+		return "", fmt.Errorf("unable to hash data: %v", err)
+	}
+
+	return fmt.Sprintf("%d", hash.Sum32()), nil
+}
```
Comment (dstiliadis) [inline policy/types.go:389]: Confused. Who is using this method? I don't see anything below.

Comment (sibicramesh) [reply policy/types.go:389]: This is used by this PR https://github.com/aporeto-inc/trireme-lib/pull/838 and as well as in enforcer which is for a a different use case. Which ever goes first the other will either be closed or conflicts will be resolved.

**Pull request closed without merging**